rewrite-core/src/main/java/org/openrewrite/Recipe.java

```diff
@@ -416,4 +420,8 @@ public Object clone() {
             throw new RuntimeException(e);
         }
     }
+
+    public abstract static class DelegatingRecipe extends Recipe {
+        public abstract Recipe getDelegate();
+    }
```

Comment:
Any reason this is an abstract class rather than an interface? Seems like it could be an interface

Reply:
Because it extends recipe, which is an abstract class. We could remove the extends ofcourse, but I thought this made more sense

Reply:
I tried the inteface, it looks a little nicer, but it does not force the DelegatingRecipe to be a Recipe itself. Not sure which I like better. What do you think? (see last commit)